DOCS/man/options.rst

```diff
@@ -5252,16 +5252,15 @@ them.
     The filter function to use when upscaling video.
 
     ``bilinear``
-        Bilinear hardware texture filtering (fastest, very low quality). This
-        is the default for compatibility reasons.
+        Bilinear hardware texture filtering (fastest, very low quality). This is
+        the default when using ``fast`` profile.
 
     ``spline36``
-        Mid quality and speed. This is the default when using ``gpu-hq``.
+        Mid quality and speed.
 
     ``lanczos``
-        Lanczos scaling. Provides mid quality and speed. Generally worse than
-        ``spline36``, but it results in a slightly sharper image which is good
-        for some content types. The number of taps can be controlled with
+        Lanczos scaling. Provides good balance between quality and performance.
```

Comment:
I have a feeling this will impact performance on low-end Intel GPUs (unless the idea is to force those users to edit their settings?) or break a bunch on average Android devices. Will try to test later.

Reply:
Low-end intel GPUs are exactly the device I expect to set `--profile=fast`, though. Anything newer than a couple of years ago should be able to handle lanczos just fine. At least, my several years old laptop does.

Reply:
2D filters are converted to LUT anyway, so their performance is similar. Polar filters are heavier, that's why it is gated behind high quality profile.
```
'spline36':      7744 frames in 4.403898 seconds => 0.568685 ms/frame (1758.44 FPS), gpu time: 1.110272 ms
'lanczos':      7344 frames in 4.151102 seconds => 0.565237 ms/frame (1769.17 FPS), gpu time: 1.106754 ms
'ewa_lanczossharp':      5936 frames in 5.804746 seconds => 0.977889 ms/frame (1022.61 FPS), gpu time: 1.864377 ms
```

Reply:
Radius 2 vs radius 3 can make a big dif for non-polar filters, mind

Reply:
for Intel: concerns resolved through IRC
```
17:29 <sfan5> okay I'm thinking this laptop shouldn't be the measure, trying ewa_lanczossharp reliably causes a gpu hang
17:30 <sfan5> in any case for 720p->1080p it does 230 FPS with bilinear, 130 with lanczos or spline36
17:30 <haasn> sfan5: what about catmull_rom
17:30 <sfan5> so it's less bad than I imagined
17:30 <sfan5> 144
17:30 <haasn> heh
17:30 <haasn> scale=lanczos cscale=catmull_rom might be a good compromise
17:30 <haasn> I'm thinking
17:30 <haasn> but for highquality I see no reason not to use lanczossharp for both
17:31 <sfan5> that combo gives 116 fps
17:31 <llyyr> yeah for highquality cscale and scale should match, but for default we can use a faster cscale
17:31 <sfan5> (test video is 4:2:0)
17:31 <llyyr> what about cscale=bilinear? or hermite
17:31 <llyyr> (hermite is not mapped yet)
17:31 <haasn> bilinear is better than hermite
17:32 <sfan5> I tested with 0.36.0 so cscale should default to bilinear, right?
17:32 <haasn> to reproduce --cscale=mitchell --cscale-param1=0 --cscale-param2=0 --cscale-radius=1
17:32 <haasn> (hermite)
17:33 <sfan5> scale=lanczos + hermite for cscale is basically also 116 fps here
17:39 <haasn> sfan5: so about halved perf on a shittop, I'm okay with that
```

for Android: depending on vendor or model drivers are broken beyond hope anyway, not relevant